src/constants/apiPaths.ts

```diff
@@ -4,7 +4,7 @@ const API_PATHS = {
   order: 'https://a7o6p63xnd.execute-api.eu-west-1.amazonaws.com/dev',
   import: 'https://k3ch3d467k.execute-api.eu-west-1.amazonaws.com/dev',
   bff: 'https://a7o6p63xnd.execute-api.eu-west-1.amazonaws.com/dev',
-  cart: 'https://a7o6p63xnd.execute-api.eu-west-1.amazonaws.com/dev',
+  cart: 'http://drdiman-cart-api-develop.eu-west-1.elasticbeanstalk.com/api',
```

Comment:
http://drdiman-cart-api-develop.eu-west-1.elasticbeanstalk.com/api returns 404
what's the reason?

Reply:
Because it doesn't exist as separate path. It's just a base url for api paths